### ISSUE TITLE
Argument 1 passed to craft\elements\db\ElementQuery::enabledForSite() must be of the type bool, null given

### ISSUE THREAD

**Issue by matthisamoto** [page-8/craft-manytomany#32]
As of Craft CMS v3.4.6.1 while using the current version of the plugin available through this repo, I am seeing an error returned when attempting to add an entry to a Many to Many field.

```
Failed to load resource: the server responded with a status of 500 (Internal Server Error)
/index.php?p=admin/actions/element-indexes/get-elements&v=1582172064925:1
```

```
error: "Argument 1 passed to craft\elements\db\ElementQuery::enabledForSite() must be of the type bool, null given, called in ~/vendor/craftcms/cms/src/elements/db/ElementQuery.php on line 516"
```

**Application Info**
PHP version	7.3.14
OS version	Linux 3.10.0-957.27.2.el7.x86_64
Database driver & version	MySQL 5.7.29
Image driver & version	GD 7.3.14
Craft edition & version	Craft Pro 3.4.6.1

**Comment by matthisamoto** [comment]
Updated because I was clicking through and the error I saw before was incorrect due to direct access of the endpoint.

**Comment by matthisamoto** [comment]
This can be corrected by updating line 10 in input.js to:

`criteria: {status: null, localeEnabled: false},`

**Comment by matthisamoto** [comment]
localeEnabled is actually deprecated so it should be:

`criteria: {status: null, enabledForSite: false},`

**Comment by engram-design** [comment]
Fixed in [2.0.0](https://github.com/verbb/many-to-many/releases/tag/2.0.0)
